streams/src/main/java/org/apache/kafka/streams/processor/internals/StateDirectory.java

```diff
@@ -280,7 +280,7 @@ synchronized void unlock(final TaskId taskId) throws IOException {
     public synchronized void clean() {
         // remove task dirs
         try {
-            cleanRemovedTasks(0, true);
+            cleanRemovedTasksCalledByUser();
         } catch (final Exception e) {
             // this is already logged within cleanRemovedTasks
```

Comment:
cleanRemovedTasks -> cleanRemovedTasksCalledByUser

Reply:
The previous comment is invalid as the method "cleanRemovedTasks" is removed. The replacement is "cleanRemovedTasksCalledByUser"